### DISCUSSION TITLE
Why did I successfully upload the new code, but the execution action on the leaderboard was the same as before

### DISCUSSION THREAD

**Discussion by Henry-Lee24** [cyberbotics/wrestling#81] [Q&A] [answered]
I uploaded it this way, and I hope you can help me see where the problem lies: I was directly uploading it to the warehouse using Git in the local competition folder, but due to an error I may have encountered somewhere, there were two branches after uploading (one is the local master and the other is the main in the warehouse). Afterwards, I deleted the main branch and renamed the master as the main branch. This way, I can successfully upload and modify the main branch, but the code executed on the leaderboard is the code from the previous upload

**Comment by omichel** [comment, marked as answer]
From what I see, [your repository](https://github.com/Henry-Lee24/LHL-bot/tree/main) was modified 4 hours ago. Right after this modification a game was run opposing your controller against Shane Copenhagen and [you lost the game](https://github.com/cyberbotics/wrestling/actions/runs/4966620713). See details [here](https://github.com/cyberbotics/wrestling/actions/runs/4966620713/jobs/8888156530).

*Note*: open all the `>` sections to see more information about how your controller lost its game.

**Reply by Henry-Lee24** [reply]
I know that my code has been updated and run on the leaderboard, but the running code is not my latest code, but the code I last modified.

**Reply by Henry-Lee24** [reply]
I have solved this problem.This is my problem, thank you.
